File: 2011-FinalValueofVariableAfterPerformingOperations/2011-FinalValueofVariableAfterPerformingOperations.py
# Last updated: 20/10/2025, 11:49:03 am
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        n = len(operations)

        x = 0
        for op in operations:
            if op == "X++" or op == "++X":
                x += 1
            elif op == "X--" or op == "--X":
                x -= 1
        
        return x

File: 2011-FinalValueofVariableAfterPerformingOperations/2011-FinalValueofVariableAfterPerformingOperations.java
// Last updated: 20/10/2025, 5:55:15 pm
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int ans = 0;
        for (String op: operations) {
            if (op.charAt(1) == '+') {
                ans += 1;
            } else {
                ans -= 1;
            }
        }
        return ans;
    }
}